data/eom.appdata.xml.in

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2014 Ryan Lerch <rlerch@redhat.com> -->
-<application>
+<component type="desktop">
   <id type="desktop">eom.desktop</id>
```

Comment:
Is the `id` entry fine?
`<id type="desktop">eom.desktop</id>` --> `<id>eom.desktop</id>`
I just compared to other applications in /usr/share/metainfo, eg. libreoffice-calc
What is written in docs?

Reply:
I just looked at the docs, and since the root-tag already contains a ``type="desktop"``, writing only ``eom.desktop`` should be sufficient; but I'm not sure whether the ``id`` tag should contain it again though...

Reply:
See https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Application.html
and
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic
```
The ID must follow a reverse-DNS scheme, consisting of {tld}.{vendor}.{product}, for example org.kde.Gwenview or com.hugski.ColorHug2. Ownership of {vendor}.{tld} in the domain name system guarantees uniqueness of IDs. 
```
Something like `<id>org.mate.Eom.desktop</id>`. Than the whole file needs to be renamed too
`org.mate.Eom.appdata.xml`
But i am not sure if that is needed.
I would test this first with using gnome-software package-manager.

Reply:
I forgot, in this case we need to edit /usr/share/application/eom.desktop too.
See gnome apps in this dir.
I suggest simply remove only `type="desktop"` from id entry.
What good is for libreoffice should be sufficient for MATE :)

Reply:
Referring to [this](https://wiki.debian.org/AppStream/Guidelines#Component_naming), naming such as ``org.mate.Eom.appdata.xml`` might not be necessary as long as it matches with the filename.

Reply:
Can you please update PR?
Than i can approve it.

Reply:
So @raveit65 are we finally doing as the following?

```suggestion
  <id>eom.desktop</id>
```
or the following:
```suggestion
  <id>org.mate.Eom.desktop</id>
```
The second requires that the .desktop file name be ``org.mate.Eom.desktop`` and this component file be ``org.mate.Eom.appdata.xml``. 

@suve  please look at this...

Reply:
First version please.
`<id>eom.desktop</id>`

Reply:
I don't seem to have push access... will I try as remote when I get back?

Reply:
Changed to `<id>eom.desktop</id>`.